.github/workflows/php.yml

```diff
@@ -0,0 +1,63 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.0"
+          - "8.1"
+
+        include:
+          - os: ubuntu-latest
+            php-version: "8.0"
+            compiler: jit
+
+          - os: ubuntu-latest
+            php-version: "8.1"
+            compiler: jit
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install PHP with extensions
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-version }}
+        extensions: memcached
+        coverage: none
+
+    - name: Setup Memcached server
+      uses: niden/actions-memcached@v7
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate --strict
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v2
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress
+
+    - name: Run tests with phpunit
```

Comment:
Was there a reason to switch to running PHPUnit before Psalm? It makes sense to me to do the static checks first.

Reply:
There was no reason. They both take trivial amounts of time, so I don't think it matters:
![image](https://user-images.githubusercontent.com/806152/144323228-4564f405-1dea-4f58-97e9-df4dd788e509.png)